alignment_check_experiment/02_umap_alignment_check.py

```diff
@@ -0,0 +1,324 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:biovectors]
+#     language: python
+#     name: conda-env-biovectors-py
+# ---
+
+# # Generate UMAP GIF for Token Evolution Progression
+
+# This notebook is designed to test a potential issue with the Word2Vec alignment algorithm. In the notebook [03_word_decade_figure_generator_2000_2020.ipynb](../word2vec_decade_experiment/03_word_decade_figure_generator_2000_2020.ipynb), words that shouldn't move a lot appear to be moving a lot within the early years (2000-2005). This led to the hypothesis that the alignment for word vectors doesn't work for the earlier years as there is a small amount of data. To see if there is an alignment issue I generated a umap plot of all tokens shared between 2000-2020 and color coded each point based on the year. Conclusion to be discussed at the bottom.
```

Comment:
This link is broken on Github for some reason. I think it tries to render whatever is at that location in the repo, but since it's a notebook it dies. Using this link should fix it:
https://github.com/greenelab/biovectors/blob/f04b9dae/word2vec_decade_experiment/03_word_decade_figure_generator_2000_2020.ipynb